views/pages/preferences-form.ejs

```diff
@@ -1,48 +1,50 @@
 <%- include('../partials/head'); %>
 <%- include('../partials/header-nav'); %>
+
 <h1>Your preferences</h1>
-<form action="/preferences" method="post">
 
-    <label for="genderSelect">Choose your gender</label>
+<form action="/preferences" method="post"> 
 
+    <label for="genderSelect">Choose your gender:</label>
+  
     <select name="genderSelect" id="genderSelect">
         <option value="">Your gender</option>
-        <option value="He">He</option>
-        <option value="She">She</option>
-        <option value="none">non-binary</option>
+        <option value="He" <%= preferences.ownGender == 'He' ? 'selected' : '' %> >He</option>
+        <option value="She" <%= preferences.ownGender == 'She' ? 'selected' : '' %> >She</option>
+        <option value="none" <%= preferences.ownGender == 'none' ? 'selected' : '' %> >non-binary</option>
     </select>
-
-    <label for="genderPreference">Choose a preferred gender</label>
-
+  
+    <label for="genderPreference">Choose a preferred gender:</label>
+  
     <select name="genderPreference" id="genderPreference">
         <option value="">Preferred gender</option>
-        <option value="He">He</option>
-        <option value="She">She</option>
-        <option value="both">both</option>
+        <option value="He" <%= preferences.preferredGender == 'He' ? 'selected' : '' %> >He</option>
+        <option value="She" <%= preferences.preferredGender == 'She' ? 'selected' : '' %> >She</option>
+        <option value="both" <%= preferences.preferredGender == 'both' ? 'selected' : '' %> >both</option>
     </select>
-
+  
     <div>
-
-        <label for="distance">Distance (km)</label>
-        <input type="number" id="distance" name="distance" min="0" max="200" step="1" placeholder="Maximum distance">
-
+  
+      <label for="distance">Distance</label>
```

Comment:
While we're working on this, what do think about making a label that explains what kind of distance distance we're talking about here. The word 'distance' on it's own doesn't explain much on its own. Also, keeping the unit of measurement (km for example) is nice explain what kind of number is expected.